plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaSslConfig.java

```diff
@@ -0,0 +1,157 @@
+                .build();
+        KafkaSslConfig expected = new KafkaSslConfig()
+                .setKeystoreLocation(keystorePath.toString())
+                .setKeystorePassword("keystore-password")
+                .setKeystoreType(KafkaKeystoreTruststoreType.JKS)
+                .setKeyPassword("key-password")
+                .setTruststoreLocation(truststorePath.toString())
+                .setTruststorePassword("truststore-password")
+                .setTruststoreType(KafkaKeystoreTruststoreType.JKS)
+                .setEndpointIdentificationAlgorithm(KafkaEndpointIdentificationAlgorithm.DISABLED);
+
+        assertFullMapping(properties, expected);
+    }
+
+    @Test
+    public void testEmptyPropertiesAreReturned()
+    {
+        KafkaSslConfig config = new KafkaSslConfig();
+        assertThat(config.getKafkaClientProperties()).isEmpty();
+    }
+
+    @Test
+    public void testAllConfigPropertiesAreContained()
+    {
+        KafkaSslConfig config = new KafkaSslConfig()
+                .setKeystoreLocation("/some/path/to/keystore")
+                .setKeystorePassword("superSavePasswordForKeystore")
+                .setKeystoreType(KafkaKeystoreTruststoreType.JKS)
+                .setKeyPassword("aSslKeyPassword")
+                .setTruststoreLocation("/some/path/to/truststore")
+                .setTruststorePassword("superSavePasswordForTruststore")
+                .setTruststoreType(KafkaKeystoreTruststoreType.JKS)
+                .setEndpointIdentificationAlgorithm(KafkaEndpointIdentificationAlgorithm.HTTPS);
+
+        Map<String, Object> securityProperties = config.getKafkaClientProperties();
+        assertThat(securityProperties.keySet().size()).isEqualTo(8);
+        // Since security related properties are all passed to the underlying kafka-clients library,
+        // the property names must match those expected by kafka-clients
+        assertThat(securityProperties)
+                .containsKeys(
+                        SSL_KEYSTORE_LOCATION_CONFIG,
+                        SSL_KEYSTORE_PASSWORD_CONFIG,
+                        SSL_KEYSTORE_TYPE_CONFIG,
+                        SSL_TRUSTSTORE_LOCATION_CONFIG,
+                        SSL_TRUSTSTORE_PASSWORD_CONFIG,
+                        SSL_TRUSTSTORE_TYPE_CONFIG,
+                        SSL_KEY_PASSWORD_CONFIG,
+                        SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG)
+                .containsValues(
+                        "/some/path/to/keystore",
+                        "superSavePasswordForKeystore",
+                        KafkaKeystoreTruststoreType.JKS.getName(),
+                        "aSslKeyPassword",
+                        "/some/path/to/truststore",
+                        "superSavePasswordForTruststore",
+                        KafkaKeystoreTruststoreType.JKS.getName(),
+                        KafkaEndpointIdentificationAlgorithm.HTTPS.getName());
```

Comment:
drop explicit size testing and use 
```
                .containsExactlyEntriesOf(ImmutableMap.of(
                        SSL_KEYSTORE_LOCATION_CONFIG, "/some/path/to/keystore",
                        ...
                )
```
so we actually test if proper key is assigned prover value.

Reply:
ImmutableMap.of() only supports up to 5 key-values and Map.of() can support up to 10 key-values. 

I wrapped it with ImmutableMap.copyOf(Map.of())